odl/util/phantom.py

```diff
@@ -679,6 +680,12 @@ def indicate_proj_axis(discr_space, scale_structures=0.5):
     return discr_space.element(phan)
 
 
+def white_noise(space):
+    """Standard gaussian noise in space, pointwise N(0, 1)"""
+    values = np.random.randn(*space.shape)
```

Comment:
There is also [`np.random.normal`](http://docs.scipy.org/doc/numpy/reference/generated/numpy.random.normal.html) which allows you to set mean and standard deviation.